### ISSUE TITLE
Mentions and Emojis: the listbox options should have an ID

### ISSUE THREAD

**Issue by afercia** [draft-js-plugins/draft-js-plugins#1076]
Splitting this out from #805 

In the Mentions and Emojis plugins, the `aria-activedescendant` attribute on the element that has role `combobox` correctly gets a new value while highlighting the options.

However, this value must be the same value of the highlighted option `ID` attribute. As far as I see all the options miss an ID.
